source/internal.d.ts

```diff
@@ -85,16 +88,6 @@ Note: This type is not the return type of the `.toString()` function.
 */
 export type ToString<T> = T extends string | number ? `${T}` : never;
 
-/**
```

Comment:
prefer `Subtract` in "math.d.ts" ?

Reply:
Sure, but should export `Subtract` type in `internal.d.ts` for backwards-compatible

Reply:
done, see [here](https://github.com/sindresorhus/type-fest/pull/748/files#diff-49eaac4718b21c2c3d9ed8024c97600050979229a82a0255cc3c538e36350131R16)